app/controllers/dashboard.js

```diff
@@ -20,6 +20,15 @@ const dashboardJobs = (jobModels) =>
       // eslint-disable-next-line no-underscore-dangle
       { statusString: i18n.__(`progression.${job.status}`) }, job));
 
+const dashboardSortOptions = (sort) => [
+    // eslint-disable-next-line no-underscore-dangle
+    { value: 'created', label: i18n.__('dashboard.sort.created'), selected: sort === 'created' },
+    // eslint-disable-next-line no-underscore-dangle
+    { value: 'updated', label: i18n.__('dashboard.sort.updated'), selected: sort === 'updated' },
+    // eslint-disable-next-line no-underscore-dangle
+    { value: 'alpha', label: i18n.__('dashboard.sort.alpha'), selected: sort === 'alpha' },
```

Comment:
why not call it `title` instead of `alpah` so we are open to extending it to other fields of the `job` in future